scripts/remove_after_use/node_preprint_es.py

```diff
@@ -0,0 +1,47 @@
+import django
+django.setup()
```

Comment:
Nit: `website.app.setup_django` is preferred, if only because it sets up logging to the console as an import side-effect (which is bad, but that's beside the point).